### DISCUSSION TITLE
Multiple Worker Storage Problem

### DISCUSSION THREAD

**Discussion by hamidihekmat** [taskforcesh/bullmq#1019] [Q&A]
# Question
## Is there anyway for me to lock multiple jobs to one worker node?

I am in a situation where I have to rely on one worker node to process multiple jobs. The worker node relies on local file storage. How do I lock multiple jobs to be processed on one worker node?

**Comment by manast** [comment]
So I guess you need to create one queue and only have one worker for that queue running on the node you want.
